app/views/physical_ticket/show.pdf.prawn

```diff
@@ -1,4 +1,4 @@
-prawn_document(filename: @file_name, page_layout: :portrait, :page_size =>'A4' ) do |pdf|
+prawn_document(filename: @file_name, page_layout: @conference.ticket_layout.to_sym, :page_size =>'A4' ) do |pdf|
```

Comment:
maybe moving `@conference.ticket_layout.to_sym` to the controller

This way you can also check that this variable is correctly assigned in the tests. :wink: